react/modules/OrderQueue.tsx

```diff
@@ -0,0 +1,9 @@
+import {
+  OrderQueueProvider,
+  useQueueStatus,
+  useOrderQueue,
+} from '@vtex/order-manager/dist/index.js'
```

Comment:
also, we shouldn't import the dist files like this, we should let the bundler use the best option (using the proper fields in `@vtex/order-manager/package.json`) and not couple our code with a "deployment implementation"

Reply:
after seeing with @wdsrocha, the issue is in the `@vtex/order-manager/package.json` file where the `"browser"` entry was pointing to an invalid file and the fix would be to just remove it from there